Sources/Screens/Tabs/TabbedViewController.swift

```diff
@@ -66,6 +80,14 @@ final class TabbedViewController: BaseViewController {
         resizeHeaderView()
     }
     
+    @IBAction private func updateAnalytics(_ sender: UISwitch) {
+        if sender.isOn {
+            analyticsCenter.analyticsPreferenceStore.hasAcceptedAnalytics = true
+        } else {
+            analyticsCenter.analyticsPreferenceStore.hasAcceptedAnalytics = false
+        }
+    }
```

Comment:
Seeing as we're just flipping between a `Bool` you can use `.toggle()` which will do that for you 🙂 

```suggestion
    analyticsCenter.analyticsPreferenceStore.hasAcceptedAnalytics?.toggle()
```